### PR TITLE
Adding pretender to Exegol-Images

### PR DESCRIPTION
**Pull request by stevesec**
# Description

> A description of your PR, what it brings or corrects. Don't forget to configure your PR to the dev branch (cf. https://exegol.readthedocs.io/en/latest/community/contributors.html)

# Related issues

> If your PR responds to an issue for a bug fix or feature request, make sure to includes references to the issues (e.g. "fixes #xxxx").

PR #294 

# Point of attention

> Things you are not sure about that deserve special attention if you have doubts or questions.

### CODE REVIEW

File: sources/install/package_ad.sh
Comment (ShutdownRepo) [inline sources/install/package_ad.sh:1]: don't forget to add the call to the install function 😉

**Review by ShutdownRepo — CHANGES_REQUESTED** [review]

File: sources/install/package_ad.sh
Comment (stevesec) [reply sources/install/package_ad.sh:1]: *slaps head*

**Review by ShutdownRepo — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
Careful, the target branch was wrong, fixed it

**Review by QU35T-code — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
@stevesec can you make push a change on the PR so that I can run the appropriate pipeline? Any change will do, synchronizing on the dev branch would be relevant for example

File: sources/assets/shells/history.d/pretender [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Pretender History.d
```
Comment (QU35T-code) [inline sources/assets/shells/history.d/pretender:1]: Please remove this comment (I know Shutdown asked you ;))

**Review by QU35T-code — CHANGES_REQUESTED** [review]

**Review by ShutdownRepo — APPROVED** [review]

**Comment by ShutdownRepo** [comment]
Thank you @stevesec ! 💪

**Comment by stevesec** [comment]
And thank you both for this amazing repo!

**Pull request merged**